bokeh/models/tiles.py

```diff
@@ -26,11 +26,11 @@ class TileSource(PlotObject):
     the maximum zoom level for the tile layer. This is the most "zoomed-in" level.
     """)
 
-    x_origin_offset = Float(default=20037508.34, help="""
+    x_origin_offset = Float(default=None, help="""
```

Comment:
Those three properties are relevant for any tile source which is why I hesitate in removing them from the base class, but in general, I do want to refactor out this class hierarchy.  Right now examples are broken in production which should be fixed ASAP (with this hot-fix).   Bigger picture though, what other mechanisms do we have to override default values thus allowing me to keep properties in base class, but provide different defaults in subclasses?